### PR TITLE
Order FullyQualify results in the same order as Add-Import

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/11240

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
test this please

**Comment by CyrusNajmabadi** [comment]
test vsi please

**Comment by CyrusNajmabadi** [comment]
@dotnet/roslyn-ide review this plz.

**Comment by CyrusNajmabadi** [comment]
retest vsi please

**Comment by CyrusNajmabadi** [comment]
retest vsi please

**Comment by CyrusNajmabadi** [comment]
retest vsi please

**Comment by CyrusNajmabadi** [comment]
retest windows_vsi_p1_open_prtest  please

File: src/Features/Core/Portable/CodeFixes/Qualify/AbstractFullyQualifyCodeFixProvider.cs [hunk 1]
```diff
@@ -235,96 +238,21 @@ private bool HasAccessibleTypes(INamespaceSymbol @namespace, SemanticModel model
...
-            symbols = symbols ?? SpecializedCollections.EmptyList<SymbolResult<INamespaceOrTypeSymbol>>();
+            symbols = symbols ?? SpecializedCollections.EmptyList<SymbolResult>();
             symbols = symbols.Distinct()
                              .Where(n => n.Symbol is INamedTypeSymbol || !((INamespaceSymbol)n.Symbol).IsGlobalNamespace)
-                             .OrderBy(Compare);
+                             .Order();
             return symbols.Select(n => n.Symbol).ToList();
         }
 
-        private static readonly ConditionalWeakTable<INamespaceOrTypeSymbol, IList<string>> s_symbolToNameMap =
```
Comment (DustinCampbell) [inline src/Features/Core/Portable/CodeFixes/Qualify/AbstractFullyQualifyCodeFixProvider.cs]: Seeing this red makes me smile.

File: src/Features/Core/Portable/CodeFixes/Qualify/AbstractFullyQualifyCodeFixProvider.cs [hunk 2]
```diff
@@ -235,96 +238,21 @@ private bool HasAccessibleTypes(INamespaceSymbol @namespace, SemanticModel model
...
-            new ConditionalWeakTable<INamespaceOrTypeSymbol, IList<string>>();
-        private static readonly ConditionalWeakTable<INamespaceOrTypeSymbol, IList<string>>.CreateValueCallback s_getNameParts = GetNameParts;
```
Comment (DustinCampbell) [inline src/Features/Core/Portable/CodeFixes/Qualify/AbstractFullyQualifyCodeFixProvider.cs]: Ditto. Another smile.

**Comment by DustinCampbell** [comment]
LGTM

**Pull request merged**